tool_path_planner/src/plane_slicer_raster_generator.cpp

```diff
@@ -514,7 +512,7 @@ boost::optional<ToolPaths> PlaneSlicerRasterGenerator::generate()
   double min_coeff = dist.minCoeff();
 
   // Calculate the number of planes to cover the bounding box along the direction vector
-  auto num_planes = static_cast<std::size_t>(std::ceil((max_coeff - min_coeff) / config_.raster_spacing));
+  std::size_t num_planes = static_cast<std::size_t>(std::ceil((max_coeff - min_coeff) / config_.raster_spacing));
```

Comment:
Should use auto for readability when casting because the type is defined in the cast no need to define it twice. I could not find the Clang-Tidy check that explains this but it will complain.

Reply:
I think it makes sense to keep the `static_cast` here because `ceil` returns a double, but `num_planes` needs to be some type of integer